#+TITLE: Basics of Biological Chemistry (Fall 2014)
#+AUTHOR: Cedric Lood
#+LATEX_CLASS: article
#+LATEX_CLASS_OPTIONS: [11pt, a4paper,titlepage]
#+LATEX_HEADER: \usepackage[left=2.35cm, right=3.35cm, top=3.35cm, bottom=3.35cm]{geometry}
#+LATEX_HEADER: \usepackage[utf8]{inputenc}
#+LATEX_HEADER: \usepackage[english]{babel}
#+LATEX_HEADER: \usepackage{graphicx}
#+LATEX_HEADER: \usepackage{titlesec}
#+LATEX_HEADER: \usepackage{chemfig}
#+LATEX_HEADER: \usepackage{tocbibind}
#+TITLE:
#+BEGIN_LaTeX
\setlength{\parskip}{0pt}%
\setlength{\parindent}{0pt}%
\renewcommand{\thesubsubsection}{\alph{subsubsection}.)}
\include{title}
\setcounter{tocdepth}{3}
\tableofcontents
\clearpage
#+END_LaTeX
#+STARTUP: latexpreview
#+OPTIONS: LaTeX:dvipng, toc:nil

* Questions from Prof. J. Vanderleyden – dr. H. Steenackers
Note: reference material for this part mostly come from the
recommended class book \cite{BioChemBlei}
** Chemical reaction equation

Consider the following reaction: (NH_{4})_{2}CO_{3} +  Zn(NO_{3})_{2} →  NH_{4}NO_{3} + ZnCO_{3}

*** Balance the equation  
(NH_{4})_{2}CO_{3} +  Zn(NO_{3})_{2} →  2 NH_{4}NO_{3} + ZnCO_{3}

*** Reactants and products
=Q:= Name all reactants and reaction products.

=A:=
- (NH_{4})_{2}CO_{3} : Ammonium Carbonate
- Zn(NO_{3})_{2} : Zinc Nitrate
- NH_{4}NO_{3} : Ammonium Nitrate
- ZnCO_{3} : Zinc Carbonate (also known as "Smithsonite")

*** Lewis structure, VESPR
=Q:= Construct the Lewis structures of the polyatomic ions you recognize
and predict their molecular structure using the VSEPR theory.

=A:=
- Lewis Structure of the ions: 
#+BEGIN_LaTeX
\renewcommand{\arraystretch}{1.5}
\begin{tabular}{ c | c | c | c}
Ammonium & Carbonate & Zinc & Nitrate  \\
\hline
\chemfig{N^{+}(-[:0]H)(-[:90]H)(-[:180]H)(-[:270]H)} &
\chemfig{\lewis{3:5:,O}=C(-[1]\lewis{3:1:7:,O}^{-})(-[7]\lewis{1:7:5:,O}^{-})} &
\chemfig{\lewis{0.4.,Zn^{2+}}} &
\chemfig{\lewis{3:5:,O}=N^{+}(-[1]\lewis{3:1:7:,O}^{-})(-[7]\lewis{1:7:5:,O}^{-})}\\
\end{tabular}
#+END_LaTeX

- Molecular structure prediction:
#+BEGIN_LaTeX
\renewcommand{\arraystretch}{1.5}
\begin{tabular}{ c | c | c | c}
Ammonium & Carbonate & Zinc & Nitrate  \\
\hline
\chemfig{N^{+}(-[2]H)(-[5]H)(<[6]H)(<:[7]H)} &
\chemfig{O=C(-[1]O^{-})(-[7]O^{-})} &
\chemfig{Zn^{2+}} &
\chemfig{O=N^{+}(-[1]O^{-})(-[7]O^{-})}\\
\end{tabular}
#+END_LaTeX

*** Oxidation states
=Q:= Determine the oxidation state of all the atoms in all the
compounds. Is this an oxidation-reduction reaction?

=A:= Ammonium Carbonate and Zinc Nitrate (the reactants) are very
soluble in water and will thus move freely as charged ions. The Zinc
and Carbonate ions will precipitate. The reaction is not an
oxydo-reduction as shown by the details below:

- Zn(NO_{3})_{2} each O are -2, N is +5, (NO3^{-})_{2} is -2, and Zn is +2
- (NH_{4})_{2}CO_{3} N is -3, each H are +1, and (NH_{4})_{2} is
     +2. C is +4 and each O is -2, and (CO_3)^{-2} is -2.
- NH_{4}NO_{3} for NH_{4}, the oxidation state is +1, and for NO_{3}
  it's -1
- ZnCO_{3} for Zn, the oxidation is +2, and it's -2 for CO_3

*** Mass
=Q:= How many grams of ZnCO3 can be prepared from 400g Zn(NO3)2 by
using sufficient(NH4)2CO3?

=A:= Let's start by computing the molecular weight of the 2 reactants:

- Molecular weigth of Zn(NO_{3})_{2} :: 189.36 g/mol
- Molecular weigth of (NH_{4})_{2}CO_{3} :: 96.09 g/mol

Given that there is 400g of Zn(NO_{3})_{2}, we can calculate the
number of moles of reactant (and ignore that of (NH_{4})_{2}CO_{3}
since it is in excess):

- Moles of Zn(NO_{3})_{2} :: 400 g / 189.36 g/mol = 2.11237 moles

From this last figure, we can infer that the number of moles of
ZnCO_{3} will be 2.11237. Given the molecular mass of ZnCO_{3}, we can
compute the amount of ZnCO_{3} produced to be: 2.11237 mol * 125.3889
g/mol = 264.87 g.

** DNA sequence analysis

The following diagram shows part of a template DNA strand, with
sections X,Y and Z being the exons of a gene:

#+BEGIN_EXAMPLE
5'                        3'
GTA GGT TGT ATC GAT GGT CAT
---     -------         ---
 X         Y             Z
#+END_EXAMPLE

*** DNA Replication
=Q:= What is the corresponding sequence on the new daughter strand
made from the given parent strand during replication?

=A:= Given the principle of base pairing, we can determine the daughter
sequence to be:

#+BEGIN_EXAMPLE
5'                        3'
GTA GGT TGT ATC GAT GGT CAT
CAT CCA ACA TAG CTA CCA GTA
3'                        5'
#+END_EXAMPLE

*** Translated Protein
=Q:= What polypeptide sequence will be synthesized from the given template
DNA? Give a short overview of the different processes (and enzymes)
involved in the synthesis of polypeptides from template DNA. Where in
the cell do these processes take place?

=A:= The synthesized polypeptide will consist of the amino acids
=Met-Asp-Thr-Tyr= (corresponding mRNA: =5'AUGGAUACAUAC=). Since it is
mentioned that exons are present, we can assume the translation will
take place with the eukaryotic machinery. It will thus consist of the
following general steps:

- Transcription: In the nucleus, the RNA Polymerase II will be
  recruited and will bind to the promoter of the gene. It will
  produce, by moving in the 5' to 3' direction, a pre-messenger RNA
  which will be identical to the DNA template sequence (with the
  exception that Uracyl will be used instead of Thymine, and also the
  addition of a 5' CAP). That messenger RNA will then be processed by
  the spliceosome, which will remove the introns, and a Poly-A tail
  will also be added at the 3' end of the mRNA. The mRNA is then
  ready to go outside of the nucleus to be translated.

- Translation: the mRNA leaves the nucleus and it will be captured by
  a ribosome that will either bind to the ER or not depending on the
  signal encoded in the mRNA. It will then start scanning for a start
  codon in the mRNA. From that point on, the synthesis of a
  polypeptide will be accomplished by reading 3 base pairs at a time
  and pairing these 3 with the correct tRNA. After that, the
  polypeptide will either be processed further and sent to the golgi
  apparatus, or will remain in the cytosol.

*** Mutated exon
=Q:= What polypeptide sequence will be synthesized if the ATC in exon
Y is mutated to TTC? What polypeptide sequence will be synthesized if
the ATC in exon Y is mutated to ATG? Which of those substitution
mutations is likely to be more harmful? Why?

=A:= Here are the new sequences with mutated exons:

- TGTATC -> TGTTTC : the resulting polypeptide will be =Met-Glu-Thr-Tyr= 
- TGTATC -> TGTATG : the resulting polypeptide will be =Met-His-Thr-Tyr=

The second mutation would be the most disruptive. Indeed the original
Aspartate would be negatively charged in the physiological condition,
and the Glutamate would also be negatively charged, the only
difference between the 2 is then an additional CH2 group in the side
chain. Histidine on the the other hand is neutral in physiological
conditions, its side chain is also significantly larger/bulkier due to
the presence of an aromatic ring.

*** Interactions with antibiotics
=Q:= Which steps in polypeptide synthesis are affected by resp. the
macrolide antibiotics and the tetracycline antibiotics?

=A:= Both substance have the capabilities to inhibit the synthesis of
proteins by affecting ribosomal activity \cite{AntibioticsRibosomeEffect}. 

- Macrolide : prevents peptidyltransferase from linking the peptide
  from the tRNA to the growing polypeptide chain.

- Tetracycline : this one functions by preventing proper binding of tRNA
  to mRNA in the ribosome. 

*** Comparison of error rates
=Q:= The error rate in RNA synthesis is much higher than the error rate
of DNA replication. What is the origin of this difference? Motivate
why this is not a serious problem.

=A:= DNA being the central repository of the genetic information for
an organism, the fidelity of the DNA replication is required to ensure
the continuity of the species and its viability accross multiple
generation. The cell needs thus enforce a high fidelity of the
replication process through an extensive proof reading system. On the
other hand, there is no proof reading for transcription. Whenever an
incorrect mRNA is transcribed, the effect are very local and
temporary. Indeed there is no real harm in producing a couple of
non-functioning mRNA or proteins that will eventually be degraded by
the cell.

** tRNA 3D-Structure
=Q:= All tRNA molecules have a particular 3D-structure. Which
functional groups and which chemical bonds/interactions contribute to
this particular structure? Why is this particular structure of
importance for the biological function?

=A:= Below is a representation of a tRNA structure
\cite{tRNA-Phe}. The structure of the tRNA contains a couple of loops
and contains parts with base pairing (hydrogen bonds). This structure
is critical for the correct capture, processing, and release of the
tRNAs by the ribosomes. A couple of important sections can be identified
which are common to tRNAs and critical to their function :

- Anticodon arm (blue): that loop will contain the anticodon (black)
  which will base pair with the mRNA codon.
- Acceptor stem (purple): which is the attachment site of the amino
  acids.
- T-Arm (green): that region is a special recognition site for the
  ribosome. It allows a tRNA-ribosome complex to form and translation
  to proceed.

#+ATTR_LATEX: width=10cm
[[./Figures/TRNA-Phe_yeast.png]]

* Questions from Prof. B. Sels
Note: reference material for this part mostly come from the
recommended class book \cite{BioChemBlei}
** Biopolymer organisation
=Q:= The course and the textbook systematically organize four important
biopolymers mainly according to their chemical structure. Attempt a
complete reorganization of the various biopolymer structures (and
subfamilies!) according to the following three physiological
functions: energy, structure, and communication. Explain the
physiological function of each biopolymer type with regard to its
chemical structure and/or physical properties.

=A:= The fours main categories of polypeptide consist of the
carbohydrates, the nucleic acids, the proteins (polypeptides), and the
lipids. Here is my attempt at reorganizing them based on the following
categories:

*** Energy
- Carbohydrates for short term storage of energy, either for immediate
  release of energy (glucose, galactose, etc), or for midterm storage
  of energy (Polysaccharide glycogen).
- Lipids can be used in a triacylglycerols form to store energy for
  the long term.
- The universal energy conveyor of the cellular life is ATP, which is
  a triphosphated adenosine (nucleic acid).
*** Structure
- Glycerophospholipids are the most abundant lipids found in the cell
  membrane. 
- Steroids such as cholesterol are important for the fluidity of the
  membrane.
- Fibrous proteins are used to form the exoskeletton and the
  misc filaments inside the cell (Actin, Tubulin, etc). They also
  play an important part of the movement capabilities for unicellular
  organism (flagella, cilia)
- Polysaccharides can be used for structure as well, for example as
  cellulose for the rigidity of the plants, or as chitin for the
  exosqueleton of the insects.
- Peptidoglycans are a major component of bacterial cell walls (they
  give it strength)
- Waxes, which are non polar esters of long-chain fatty acids and
  long chain of monohydroxylic alcoohols are widely distributed in
  nature as protective, waterproof coatings on leaves, fruits, animal
  skin, fur, and feathers.
*** Communication
- DNA is the central repository of information for the
  organism genetic makeup, which is passed on through the
  generations.
- RNA is the intermediate messenger for the translation of
  proteins. 
- Modified amino acids are involved in communication too (eg Thyroxine
  and Melatonin). Important in this category are the cAMP, and cGMP
  which are often found as secondary messenger units for
  communication within the cell.
- A host of membrane proteins that help recognizing and process
  signals are found on most cells. An important family of these
  membrane proteins are the G-Proteins which help with signal
  transduction. 
- Neurotransmitters that carry signal from one neurons' synapses to
  another. Many of these neurotransmitters are synthesized from amino
  acids precursors.

** Chemical structure of proteins and proteins separation
=Q:= Draw the chemical structure of the following two oligopeptide
structures, a) Gln-Ser-Lys-Lys-Ser and b) Cys-Asp-Asp-Glu-Lys,
determine its net charge in physiological conditions. How would you
separate the two peptides ?  

=A:= These are the chemical structures of:
- Gln-Ser-Lys-Lys-Ser

  #+BEGIN_LaTeX
  \setatomsep{25pt}
  \chemfig{NH3^{+}-C(-[2]H)(-[6]CH2(-[6]CH2(-[6]C(=[7]O)(-[5]NH2))))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6]CH2(-[6]OH))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6](CH2(-[6]CH2(-[6]CH2(-[6]CH2(-[6]NH3^{+}))))))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6](CH2(-[6]CH2(-[6]CH2(-[6]CH2(-[6]NH3^{+}))))))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6]CH2(-[6]OH))-COO^{-}}
  #+END_LaTeX

- Cys-Asp-Asp-Glu-Lys

  #+BEGIN_LaTeX
  \setatomsep{25pt}
  \chemfig{NH3^{+}-C(-[2]H)(-[6]CH2(-[6]SH))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6]CH2(-[6]COO^{-}))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6]CH2(-[6]COO^{-}))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6]CH2(-[6]CH2(-[6]COO^{-})))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6](CH2(-[6]CH2(-[6]CH2(-[6]CH2(-[6]NH3^{+}))))))-COO^{-}}
  #+END_LaTeX

Under physiological conditions (ie, pH around 7.35), these would be
the net charge on each polypeptide:

- Gln-Ser-Lys-Lys-Ser: net charge is +2

  #+BEGIN_LaTeX
  \chemfig{\chemabove{NH3}{\scriptstyle\oplus}-Gln-Ser-\chemabove{Lys}{\scriptstyle\oplus}-\chemabove{Lys}{\scriptstyle\oplus}-Ser-\chemabove{COO}{\ominus}}
  #+END_LaTeX

- Cys-Asp-Asp-Glu-Lys: net charge is -2

  #+BEGIN_LaTeX
  \chemfig{\chemabove{NH3}{\scriptstyle\oplus}-Cys-\chemabove{Asp}{\ominus}-\chemabove{Asp}{\ominus}-\chemabove{Glu}{\ominus}-\chemabove{Ly}{\scriptstyle\oplus}-\chemabove{COO}{\ominus}}
  #+END_LaTeX

Separation of both proteins can thus be achieved by ion exchange
chromatography since they both have quite a distinct charge
\cite{BioChemPrinciples}. For example, we could use anion exchange
chromatography, which would capture the negatively charged
polypeptide, and let the positively charged one pass through. The
former could then be recovered later on by washing away the column.

** Chemical structure of disaccharides
=Q:= Draw the chemical structure of the following disaccharides: a)
the \beta-anomer of \alpha(1→6)galactoglucose and b)
\beta,\alpha(1→2)glucofructose.

=A:= These are the chemical structure of:
- \beta-anomer of \alpha(1→6)galactoglucose

Beta anomers have a cis relationship between the CH_{2}OH group on the
C_{1} and the OH group on the C_{6}. This helps us determine the
structure of the monosaccharide glucose. The polymerisation is
achieved through an \alpha binding between the C_{6} of the Galactose,
and the C_{1} of the glucose molecule, giving the following molecular
structure:

#+ATTR_LATEX: width=9cm
[[./Figures/B-A(1-6)GalactoGlucose.pdf]]

- \beta,\alpha(1→2)glucofructose

The same principle are applied here for the dissacharide consisting
of Glucose and fructose:

#+ATTR_LATEX: width=6cm
[[./Figures/BA(1-2)GlucoFructose2.pdf]]

* Questions from Prof. D. De Vos
Note: reference material for this part mostly come from the
recommended class book \cite{BioChemBlei}

Considering the following molecule:

#+ATTR_LATEX: width=8cm
[[./Figures/Part3MoleculeRaw.pdf]]

** Functional groups
=Q:= Name all functional groups

=A:= See annoted figure below

#+ATTR_LATEX: width=11cm
[[./Figures/Part3MoleculeFunctionalGroups.pdf]]

** Water and oil solubility factors
=Q:= Indicate which groups make the molecule rather water-soluble
than oil-soluble

=A:= The following groups can partake in hydrogen bonds with water
molecules and increase the solubility of the molecule in water :

- Hydroxyl groups (5 of them)
- Carbonyl groups (3 of them)
- Amino group (1 present)

We can also observe that the molecule (which is the molecule of
Doxorubicin, used among other things for treatment of cancer via
chemotherapy \cite{Doxorubicin}) contains interesting linkages, most
notably ether bonds, of which the electron pairs on the oxygens can
partake in H-Bonds with water and thus make the molecule more
hydropholic.

#+LATEX: \vfill
#+LATEX: \bibliographystyle{plain}
#+LATEX: \bibliography{bib-db}
