#+TITLE: Basics of Biological Chemistry (Fall 2014)
#+AUTHOR: Cedric Lood
#+LATEX_CLASS: article
#+LATEX_CLASS_OPTIONS: [11pt, a4paper,titlepage]
#+LATEX_HEADER: \usepackage[left=2.35cm, right=3.35cm, top=3.35cm, bottom=3.35cm]{geometry}
#+LATEX_HEADER: \usepackage[utf8]{inputenc}
#+LATEX_HEADER: \usepackage[english]{babel}
#+LATEX_HEADER: \usepackage{graphicx}
#+LATEX_HEADER: \usepackage{titlesec}
#+LATEX_HEADER: \usepackage{chemfig}
#+TITLE:
#+BEGIN_LaTeX
\setlength{\parskip}{0pt}%
\setlength{\parindent}{0pt}%
\renewcommand{\thesubsubsection}{\alph{subsubsection}.)}
\include{title}
\setcounter{tocdepth}{3}
\tableofcontents
\clearpage
#+END_LaTeX
#+STARTUP: latexpreview
#+OPTIONS: LaTeX:dvipng, toc:nil



* Questions from Prof. J. Vanderleyden – dr. H. Steenackers
** Chemical reaction equation

Consider the following reaction: (NH_{4})_{2}CO_{3} +  Zn(NO_{3})_{2} →  NH_{4}NO_{3} + ZnCO_{3}

*** Balance the equation  
(NH_{4})_{2}CO_{3} +  Zn(NO_{3})_{2} →  2 NH_{4}NO_{3} + ZnCO_{3}

*** Reactants and products
=Q:= Name all reactants and reaction products.

=A:=
- (NH_{4})_{2}CO_{3} : Ammonium carbonate
- Zn(NO_{3})_{2} : Zinc nitrate
- NH_{4}NO_{3} : Ammonium nitrate
- ZnCO_{3} : Zinc carbonate

*** Lewis structure, VESPR
=Q:= Construct the Lewis structures of the polyatomic ions you recognize
and predict their molecular structure using the VSEPR theory.

=A:=
- Lewis Structure of the ions: 
#+BEGIN_LaTeX
\renewcommand{\arraystretch}{1.5}
\begin{tabular}{ c | c | c | c}
Ammonium & Carbonate & Zinc & Nitrate  \\
\hline
\chemfig{N^{+}(-[:0]H)(-[:90]H)(-[:180]H)(-[:270]H)} &
\chemfig{\lewis{3:5:,O}=C(-[1]\lewis{3:1:7:,O}^{-})(-[7]\lewis{1:7:5:,O}^{-})} &
\chemfig{\lewis{4:,Zn^{2+}}} &
\chemfig{\lewis{3:5:,O}=N^{+}(-[1]\lewis{3:1:7:,O}^{-})(-[7]\lewis{1:7:5:,O}^{-})}\\
\end{tabular}
#+END_LaTeX

- Molecular structure prediction:
#+BEGIN_LaTeX
\renewcommand{\arraystretch}{1.5}
\begin{tabular}{ c | c | c | c}
Ammonium & Carbonate & Zinc & Nitrate  \\
\hline
\chemfig{N^{+}(-[2]H)(-[5]H)(<[6]H)(<:[7]H)} &
\chemfig{O=C(-[1]O^{-})(-[7]O^{-})} &
\chemfig{Zn^{2+}} &
\chemfig{O=N^{+}(-[1]O^{-})(-[7]O^{-})}\\
\end{tabular}
#+END_LaTeX

*** Oxidation states
=Q:= Determine the oxidation state of all the atoms in all the
compounds. Is this an oxidation-reduction reaction?

=A:= Ammonium Carbonate and Zinc Nitrate (the reactants) are very soluble
in water and will thus move freely. The Zinc and Carbonate ions will
then precipitate.

- Zn has an oxidation state of 2:  Zn →  Zn^{2+} + 2 e^{-}
- (CO_{3}) has an oxidation state of 4: (CO_{3})^{2-} + 2 e^{-} → CO_{3}


*** Mass
=Q:= How many grams of ZnCO3 can be prepared from 400g Zn(NO3)2 by
using sufficient(NH4)2CO3?

=A:= Let's start by computing the molecular weight of the 2 reactants:

- Molecular weigth of Zn(NO_{3})_{2} :: 189.36 g/mol
- Molecular weigth of (NH_{4})_{2}CO_{3} :: 96.09 g/mol

Given that there are 400g of Zn(NO_{3})_{2}, we can calculate the
number of moles of reactant (and ignore that of (NH_{4})_{2}CO_{3}
since it is in excess):

- Moles of Zn(NO_{3})_{2} :: 400 g / 189.36 g/mol = 2.11237 moles

From this last figure, we can infer that the number of moles of
ZnCO_{3} will be 2.11237. Given the molecular mass of ZnCO_{3}, we can
compute the amount of ZnCO_{3} produced to be: 2.11237 mol * 125.3889
g/mol = 264.8678 g.

** DNA sequence analysis

The following diagram shows part of a template DNA strand, with
sections X,Y and Z being the exons of a gene:

#+BEGIN_EXAMPLE
5'                        3'
GTA GGT TGT ATC GAT GGT CAT
---     -------         ---
 X         Y             Z
#+END_EXAMPLE

*** DNA Replication
=Q:= What is the corresponding sequence on the new daughter strand
made from the given parent strand during replication?

=A:= Given the principle of base pairing, we can determine the daughter
sequence to be (here in the 3' to 5' direction):

#+BEGIN_EXAMPLE
5'                        3'
GTA GGT TGT ATC GAT GGT CAT
CAT CCA ACA TAG CTA CCA GTA
3'                        5'
#+END_EXAMPLE

*** Translated Protein
=Q:= What polypeptide sequence will be synthesized from the given template
DNA? Give a short overview of the different processes (and enzymes)
involved in the synthesis of polypeptides from template DNA. Where in
the cell do these processes take place?

=A:= The synthesized polypeptide will consist of the amino acids
=Val-Cys-Ile-His=. Since it is mentioned that exons are present, we
can assume the translation will take place with the eukaryotic
machinery. It will consist of the following stages:

- Transcription: In the nucleus, the RNA Polymerase II will be
  recruited and will bind to the promoter of the gene. It will
  produce, by moving in the 5' to 3' direction, a pre messenger RNA
  which will be identical to the DNA template sequence (with the
  exception that Uracyl will be used instead of Thymine, and also the
  addition of a 5' CAP). That messenger RNA will then be processed by
  the spliceosome, which will remove the introns, and a Poly-A tail
  will also be added at the 3' end of the mRNA. The mRNA is then
  ready to go outside of the nucleus to be translated.

- Translation: the mRNA leaves the nucleus and passes through the
  reticulated ER where it will be captured by a ribosome that will
  either bind to the ER or not. It will then start scanning for a
  start codon in the mRNA. From that point on, the synthesis of a
  polypeptide will be accomplished by reading 3 base pairs at a time
  and pairing these 3 with the correct tRNA. After that, the
  polypeptide will either be processed further and sent to the golgi
  apparatus, or will remain in the cytosol.

*** Mutated exon
=Q:= What polypeptide sequence will be synthesized if the ATC in exon
Y is mutated to TTC? What polypeptide sequence will be synthesized if
the ATC in exon Y is mutated to ATG? Which of those substitution
mutations is likely to be more harmful? Why?

=A:= Here are the new sequences with mutated exons:

- TGTATC -> TGTTTC : the resulting polypeptide will be =Val-Phe-His= 
-  -> ATG : the resulting polypeptide will be =Val-Met-His=

The second
*** Interactions with antibiotics
=Q:= Which steps in polypeptide synthesis are affected by resp. the
macrolide antibiotics and the tetracycline antibiotics?

=A:=
*** Comparison of error rates
=Q:= The error rate in RNA synthesis is much higher than the error rate
of DNA replication. What is the origin of this difference? Motivate
why this is not a serious problem.

=A:=
** tRNA 3D-Structure
=Q:= All tRNA molecules have a particular 3D-structure. Which
functional groups and which chemical bonds/interactions contribute to
this particular structure? Why is this particular structure of
importance for the biological function?

=A:=
* Questions from Prof. B. Sels
** Biopolymer organisation
=Q:= The course and the textbook systematically organize four important
biopolymers mainly according to their chemical structure. Attempt a
complete reorganization of the various biopolymer structures (and
subfamilies!) according to the following three physiological
functions: energy, structure, and communication. Explain the
physiological function of each biopolymer type with regard to its
chemical structure and/or physical properties.

=A:=
** Chemical structure of proteins and proteins separation
=Q:= Draw the chemical structure of the following two oligopeptide
structures, a) Gln-Ser-Lys-Lys-Ser and b) Cys-Asp-Asp-Glu-Lys,
determine its net charge in physiological conditions. How would you
separate the two peptides ?  

=A:= These are the chemical structures of:
- Gln-Ser-Lys-Lys-Ser

  #+BEGIN_LaTeX
  \setatomsep{25pt}
  \chemfig{NH3^{+}-C(-[2]H)(-[6]CH2(-[6]CH2(-[6]C(=[7]O)(-[5]NH2))))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6]CH2(-[6]OH))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6](CH2(-[6]CH2(-[6]CH2(-[6]CH2(-[6]NH3^{+}))))))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6](CH2(-[6]CH2(-[6]CH2(-[6]CH2(-[6]NH3^{+}))))))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6]CH2(-[6]OH))-COO^{-}}
  #+END_LaTeX

- Cys-Asp-Asp-Glu-Lys

  #+BEGIN_LaTeX
  \setatomsep{25pt}
  \chemfig{NH3^{+}-C(-[2]H)(-[6]CH2(-[6]SH))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6]CH2(-[6]COO^{-}))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6]CH2(-[6]COO^{-}))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6]CH2(-[6]CH2(-[6]COO^{-})))-C(=[2]O)-N(-[6]H)-C(-[2]H)(-[6](CH2(-[6]CH2(-[6]CH2(-[6]CH2(-[6]NH3^{+}))))))-COO^{-}}
  #+END_LaTeX

Under physiological conditions (ie, pH around 7.35), these would be
the net charge on each polypeptide:

- Gln-Ser-Lys-Lys-Ser: net charge is +2

  #+BEGIN_LaTeX
  \chemfig{\chemabove{NH3}{\scriptstyle\oplus}-Gln-Ser-\chemabove{Lys}{\scriptstyle\oplus}-\chemabove{Lys}{\scriptstyle\oplus}-Ser-\chemabove{COO}{\ominus}}
  #+END_LaTeX

- Cys-Asp-Asp-Glu-Lys: net charge is -2

  #+BEGIN_LaTeX
  \chemfig{\chemabove{NH3}{\scriptstyle\oplus}-Cys-\chemabove{Asp}{\ominus}-\chemabove{Asp}{\ominus}-\chemabove{Glu}{\ominus}-\chemabove{Ly}{\scriptstyle\oplus}-\chemabove{COO}{\ominus}}
  #+END_LaTeX

Separation of both proteins can thus be achieved by ion exchange
chromatography since they both have a distinct charge.

** Chemical structure of disaccharides
=Q:= Draw the chemical structure of the following disaccharides: a)
the \beta-anomer of \alpha(1→6)galactoglucose and b)
\beta,\alpha(1→2)glucofructose.

=A:= These are the chemical structure of:
- \beta-anomer of \alpha(1→6)galactoglucose

Beta anomers have a cis relationship between the CH_{2}OH group on
the C_{1} and the OH group on the C_{6}. This helps us determine the
structure of the monosaccharides galactose and glucose. The
polymerisation is achieved through an \alpha binding between the
C_{6} of the Galactose, and the C_{1} of the glucose molecule, giving
the following molecular structure:

#+ATTR_LATEX: width=11cm
[[./Figures/B-A(1-6)GalactoGlucose.pdf]]

- \beta,\alpha(1→2)glucofructose

#+ATTR_LATEX: width=8cm
[[./Figures/BA(1-2)GlucoFructose2.pdf]]


* Questions from Prof. D. De Vos
Considering the following molecule:

#+ATTR_LATEX: width=10cm
[[./Figures/Part3MoleculeRaw.pdf]]

** Functional groups
=Q:= Name all functional groups

=A:= See annoted figure below

#+ATTR_LATEX: width=11cm
[[./Figures/Part3MoleculeFunctionalGroups.pdf]]

** Water and oil solubility factors
=Q:= Indicate which groups make the molecule rather water-soluble
than oil-soluble

=A:= The following groups can partake in hydrogen bonds with water
molecules and increase the solubility of the molecule in water :

- Hydroxyl groups (5 of them)
- Carbonyl groups (3 of them)
- Amino group (1 present)
